src/main/kotlin/uk/gov/justice/digital/hmpps/hmppschallengesupportinterventionplanapi/service/DecisionActionsService.kt

```diff
@@ -33,24 +34,16 @@ class DecisionActionsService(
     return with(verifyExists(record.referral) { MissingReferralException(recordUuid) }) {
       csipRecordRepository.save(
         createDecisionAndActions(
+          context = context,
           decisionOutcome = decisionOutcome,
           decisionOutcomeSignedOffBy = decisionOutcomeSignedOffBy,
           decisionConclusion = request.conclusion,
-          decisionOutcomeRecordedBy = context.username,
```

Comment:
The intention now is to assign decisionOutcomeRecordedBy, decisionOutcomeRecordedByDisplayName and decisionOutcomeDate from the properties in CreateDecisionAndActionsRequest. Which I now see are outcome rather than decisionOutcome. They can all be shortened to remove outcome/decisionOutcome